lib/ASRouterTargeting.jsm

```diff
@@ -141,7 +143,12 @@ const TargetingGetters = {
 this.ASRouterTargeting = {
   Environment: TargetingGetters,
 
-  isMatch(filterExpression, target, context = this.Environment) {
+  ERROR_TYPES: {
+    MALFORMED_EXPRESSION: "MALFORMED_EXPRESSION",
+    OTHER_ERROR: "OTHER_ERROR"
+  },
+
+  isMatch(filterExpression, context = this.Environment) {
```

Comment:
So we don't need the `target` param anymore? I guess it wasn't being used... why did we have it in the first place?

Reply:
https://github.com/mozilla/activity-stream/commit/1a74d11862da63152216a215f8f8bf21349fb73c#diff-60f0c109d5c368ec6a6b11e2f311b3ef